app/assets/javascripts/components/student/student_contact.es6.jsx

```diff
@@ -20,7 +20,7 @@ class StudentContact extends Component {
         flex: '1',
         padding: '12px',
       },
-      heading: {
+      content: {
```

Comment:
Since it looks like we're not using this style we can just delete it for now (you essentially just refactored this component!).